<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>ANNEHEIM Geoffrey</author>
 <class>DialogPreferences</class>
 <widget class="QDialog" name="DialogPreferences">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>362</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Preferences Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Menu/PreferencesIcon</normaloff>:/Menu/PreferencesIcon</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabConnection">
      <attribute name="title">
       <string>Server parameters</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxServerConnection">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>Connection parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="lblServerAddress">
              <property name="minimumSize">
               <size>
                <width>46</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Address :</string>
              </property>
              <property name="buddy">
               <cstring>textProxyHostName</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textServerAddress">
              <property name="toolTip">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblServerPort">
              <property name="text">
               <string>Port :</string>
              </property>
              <property name="buddy">
               <cstring>spinBoxProxyHostPort</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxServerPort">
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>8080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxServerAutoConnect">
              <property name="text">
               <string>Auto connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProxy">
      <attribute name="title">
       <string>Proxy configuration</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxProxyHost">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="title">
          <string>General settings</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="lblProxyHostName">
              <property name="minimumSize">
               <size>
                <width>46</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Host :</string>
              </property>
              <property name="buddy">
               <cstring>textProxyHostName</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textProxyHostName">
              <property name="toolTip">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblProxyHostPort">
              <property name="text">
               <string>Port :</string>
              </property>
              <property name="buddy">
               <cstring>spinBoxProxyHostPort</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxProxyHostPort">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>3128</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxProxyAuthentification">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Authentification</string>
         </property>
         <layout class="QGridLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="lblProxyUserName">
            <property name="text">
             <string>Username :</string>
            </property>
            <property name="buddy">
             <cstring>textProxyUserName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="textProxyUserName"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblProxyUserPassword">
            <property name="text">
             <string>Password :</string>
            </property>
            <property name="buddy">
             <cstring>textProxyUserPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="textProxyUserPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxProxyUse">
           <property name="text">
            <string>Use a proxy server</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>textServerAddress</tabstop>
  <tabstop>spinBoxServerPort</tabstop>
  <tabstop>checkBoxServerAutoConnect</tabstop>
  <tabstop>textProxyHostName</tabstop>
  <tabstop>spinBoxProxyHostPort</tabstop>
  <tabstop>textProxyUserName</tabstop>
  <tabstop>textProxyUserPassword</tabstop>
  <tabstop>checkBoxProxyUse</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxProxyUse</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxProxyHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxProxyUse</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxProxyAuthentification</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
